core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -463,13 +436,13 @@ private function getLocales(): array
     {
         $dirs = [__DIR__.'/../Resources/contao/languages'];
 
-        if (is_dir($this->projectDir.'/contao/languages')) {
-            $dirs[] = $this->projectDir.'/contao/languages';
+        if (is_dir(Path::join($this->projectDir, 'contao/languages'))) {
+            $dirs[] = Path::join($this->projectDir, 'contao/languages');
```

Comment:
```suggestion
        if (is_dir($path = Path::join($this->projectDir, 'contao/languages'))) {
            $dirs[] = $path;
```

I haven't tested this suggestion! It is just a reminder because your wrote
> I introduced variables wherever the same path was used more than once.

The same applies below.